app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -1844,42 +1766,30 @@ object TelemetryWrapper {
     }
 
     @TelemetryDoc(
-            name = "Cancel FindInPage - others",
+            name = "Cancel FindInPage",
             action = Category.ACTION,
             method = Method.CANCEL,
             `object` = Object.FIND_IN_PAGE,
             value = "",
-            extras = [TelemetryExtra(name = Extra.SOURCE, value = "other"), TelemetryExtra(name = Extra.VERSION, value = "2")])
+            extras = [TelemetryExtra(name = Extra.SOURCE, value = "other,close_button,back_button"), TelemetryExtra(name = Extra.VERSION, value = "2")])
     internal fun cancelFindInPageOther() =
             EventBuilder(Category.ACTION, Method.CANCEL, Object.FIND_IN_PAGE, null)
                     .extra(Extra.SOURCE, "other")
                     .extra(Extra.VERSION, Integer.toString(FIND_IN_PAGE_VERSION))
 
-    @TelemetryDoc(
-            name = "Cancel FindInPage using close button",
-            action = Category.ACTION,
-            method = Method.CANCEL,
-            `object` = Object.FIND_IN_PAGE,
-            value = "",
-            extras = [TelemetryExtra(name = Extra.SOURCE, value = "close_button"), TelemetryExtra(name = Extra.VERSION, value = "2")])
+    // TelemetryDoc is included in cancelFindInPageOther()
     internal fun cancelFindInPageClose() =
```

Comment:
Can we refactor these too, given many are already refactored.

Reply:
fixed